src/test/resources/overseas_entity_v_1_0.json

```diff
@@ -110,6 +110,18 @@
       ],
       "non_legal_firm_members_nature_of_control_types" : [
 
+      ],
```

Comment:
We shouldn't be updating older versions of the test JSON files. Only the most recent one.